ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -225,6 +230,22 @@ public void SetZoomScale(ScrollViewer view, bool? enabled)
         }
 
         /// <summary>
+        /// Enables or disables scroll view paging mode.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="enabled">Signals whether paging is enabled.</param>
+        [ReactProp("pagingEnabled")]
```

Comment:
Any reason to use ScrollView with pagingEnabled instead of the FlipView we offer in UWP? It's pretty trivial with React Native to customize components per platform.

Reply:
FlipView is analogous to ViewPager on Android, which I believe is the recommended approach for paging there as well.

---
In reply to: [151488708](https://github.com/Microsoft/react-native-windows/pull/1518#discussion_r151488708) [](ancestors = 151488708)